.changeset/moody-seas-promise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': minor
```

Comment:
I think this is patch, but it's not super important, since another patch is coming.

Reply:
Updated to patch!